datafusion/core/src/physical_plan/hash_utils.rs

```diff
@@ -43,11 +43,11 @@ fn hash_null(random_state: &RandomState, hashes_buffer: &'_ mut [u64], mul_col:
     if mul_col {
         hashes_buffer.iter_mut().for_each(|hash| {
             // stable hash for null value
-            *hash = combine_hashes(i128::get_hash(&1, random_state), *hash);
+            *hash = combine_hashes(random_state.hash_one(&1), *hash);
```

Comment:
These changes are required due to https://github.com/tkaitchuck/aHash/pull/121 which appears to have made `CallHasher` crate private

Reply:
Largely the same change @Dandandan  applied in https://github.com/apache/arrow-rs/pull/2410